eng/pipelines/runtimelab-official.yml

```diff
@@ -64,7 +64,7 @@ extends:
             - windows_x64
             jobParameters:
               templatePath: 'templates-official'
-              buildArgs: -s tools+libs -allConfigurations -c $(_BuildConfig) /p:TestAssemblies=false /p:TestPackages=true
+              buildArgs: -s tools+libs -allConfigurations -c $(_BuildConfig) /p:BuildAllConfigurations=true /p:TestAssemblies=false /p:TestPackages=true
```

Comment:
why don't we pass -pack here?

Reply:
Because libraries packages shouldn't be generated for libraries packages in runtimelab: https://github.com/dotnet/runtime/blame/e65457031da52f57d03da721f381a82a5be64c03/eng/packaging.targets#L31